dummy-data/competitions.js

```diff
@@ -0,0 +1,27 @@
+const competitions = [
+  {
+    id: "ok79bctyuii",
+    kynologiqueDepartment: "Gdynia",
+    competitionName: "Gdanskie Zawody o Piertuszke",
+    date: new Date(2022, 3, 22),
+    place: "Gdańsk",
+    manager: "Jagienka Krzywobroda",
+    judge: "Alberto Makkaroni",
+    steward: "Wiesio Kapusta",
+    secretary: "Antonina Antioniuk",
+  },
+
+  {
+    id: "ft64dfgjhy8",
```

Comment:
I guess we should also add classes and exercises? This would be connected with storing the contestants ids, or basic data, depending on the way you decide to store it.